multihomes.dsc

```diff
@@ -0,0 +1,214 @@
+mutehome_command:
+  type: command
+  name: mutehome
+  description: used to mute specific messages in home commands
+  usage: /mutehome (command name)
+  debug: false
+  tab completions:
+    1: <list[home|delhome|sethome]>
+  script:
+    - choose <context.args.first>:
+      - case home:
+        - if <player.flag[mutehomes.home]||false>:
+          - narrate "the messages that used to appear when doing /home will appear again" format:zc_home_text
+          - flag player mutehomes.home:false
+        - else:
+          - narrate "The messages that appear when doing /home will no longer appear" format:zc_home_text
+          - flag player mutehomes.home:true
+      - case delhome:
+        - if <player.flag[mutehomes.delhome]||false>:
+          - narrate "the messages that used to appear when doing /delhome will appear again" format:zc_home_text
+          - flag player mutehomes.delhome:false
+        - else:
+          - narrate "The messages that appear when doing /delhome will no longer appear" format:zc_home_text
+          - flag player mutehomes.delhome:true
+      - case sethome:
+        - if <player.flag[mutehomes.sethome]||false>:
+          - narrate "the messages that used to appear when doing /sethome will appear again" format:zc_home_text
+          - flag player mutehomes.sethome:false
+        - else:
+          - narrate "The messages that appear when doing /sethome will no longer appear" format:zc_home_text
+          - flag player mutehomes.sethome:true
```

Comment:
these are just weird methods of checking if a player has a flag or not and forcing extra booleans
```suggestion
        - if <player.has_flag[mutehomes.home]>:
          - narrate "the messages that used to appear when doing /home will appear again" format:zc_home_text
        - else:
          - narrate "The messages that appear when doing /home will no longer appear" format:zc_home_text
          - flag player mutehomes.home
      - case delhome:
        - if <player.has_flag[mutehomes.delhome]>:
        - else:
          - narrate "The messages that appear when doing /delhome will no longer appear" format:zc_home_text
          - flag player mutehomes.delhome
      - case sethome:
        - if <player.has_flag[mutehomes.sethome]>:
          - narrate "the messages that used to appear when doing /sethome will appear again" format:zc_home_text
        - else:
          - narrate "The messages that appear when doing /sethome will no longer appear" format:zc_home_text
          - flag player mutehomes.sethome
```